packages/back-end/src/models/WebhookModel.ts

```diff
@@ -19,6 +19,13 @@ const webhookSchema = new mongoose.Schema({
   lastSuccess: Date,
   error: String,
   created: Date,
+  useSDKMode: Boolean,
```

Comment:
let me know if we should make this an ENUM like:
```
enum Modes {
CLASSIC: 0
SDK: 1
}
---------
mode: number
```

Reply:
i think its fine as is